src/hooks/useRecentFiles.jsx

```
@@ -1,59 +1,106 @@
+  }, [dataProxy])
+
+  if (!client) {
+    return {
+      data: [],
+      fetchStatus: 'error',
+      error: new Error('Client not available')
+    }
+  }
+
+  // Proxy path: merge proxy data with real-time store state
+  if (dataProxy.dataProxyServicesAvailable && proxyState.fetchStatus !== 'error') {
```

ESLint

[error] Replace `dataProxy.dataProxyServicesAvailable·&&·proxyState.fetchStatus·!==·'error'` with `⏎····dataProxy.dataProxyServicesAvailable·&&⏎····proxyState.fetchStatus·!==·'error'⏎··`

(prettier/prettier)